pages/Videos.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import Navbar from '../components/Navbar'
+import { videoURLS } from '../data/videoURLs'
+import styles from '../styles/Video.module.css'
+
+const Videos = () => {
+
+	return (
+		<div>
+			<Navbar/>
+			<div className={`${styles.videosContainer} wrapper`}>
```

Comment:
lets remove the `styles.` syntax cos of the camelCase css classes we would rather use `videos-container`

Reply:
alright agreed i'll update that

Reply:
i'm an idiot. Idk what i thought tailwind was but i'm updating it now to full use tailwind.